rosa_tutorials/rosa-mobb-byo-vpc-tutorial.adoc

```diff
@@ -0,0 +1,56 @@
+:_content-type: ASSEMBLY
+[id="rosa-mobb-byo-vpc-tutorial"]
+= Tutorial: Creating a Public/Private BYO VPC for ROSA
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-mobb-byo-vpc-tutorial
+
+//Mobb content metadata
+//Brought into ROSA product docs 2023-09-19
+//---
+//date: '2021-06-14'
+//title: Creating a Public/Private BYO VPC for ROSA
+//tags: ["AWS", "ROSA"]
+//authors:
+//  - Connor Wooley
+//---
+
+include::snippets/mobb-support-statement.adoc[leveloffset=+1]
+
+This is Terraform example to create a single AZ VPC in which to deploy a single AZ ROSA cluster. 
+
+[IMPORTANT]
+====
+This is intended to be used as a guide to get started quickly, not to be used in production.
+====
+
+== Prerequisites
+
+* link:https://www.terraform.io/downloads.html[Terraform]
+
+== Deploy
+
+. Download the following repo:
++
+[source,terminal]
+----
+$ git clone https://github.com/rh-mobb/documentation.git
+$ cd documentation/docs/rosa/byo-vpc
+----
++
+. Modify `main.tf as` needed, then run:
```

Comment:
for Documentation I expect more details how to prepare main.tf.
Will it be done in another place?
I can't see the exact main.tf so I'm not sure how clear it will be without any comment